param/__init__.py

```diff
@@ -2040,3 +2040,16 @@ def _validate(self, val):
         # values are numeric, which is redundant, so just call
         # _checkBounds().
         self._checkBounds(val)
+
+
+class Trigger(Boolean):
```

Comment:
Proposed docstring, assuming both sets of changes are kept (Trigger and .trigger):

```suggestion
class Trigger(Boolean):
    """
    A Trigger Parameter is one whose value is intimately linked to the triggering of
    events for watchers to consume. Trigger has a Boolean value, which when set to
    True triggers events sent to watchers (as any Parameter does) and then is
    automatically set back to False. Conversely, if events are triggered directly via
    `.trigger`, the value is transiently set to True (so that it's clear which of many
    parameters being watched may have changed), then restored to False when the
    triggering completes. Trigger is thus like a momentary switch or pushbutton with
    a transient True value that serves only to launch some other action (e.g. via a
    param.depends decorator), rather than encapsulating the action itself as
    param.Action does.
"""
```

Reply:
Used this as a template for the docstring pushed in af0a5d8